tests/fixtures/payload.py

```diff
@@ -146,6 +148,17 @@
 }
 
 
+start_datetime = datetime(2000, 1, 1)
+end_datetime = datetime(2001, 1, 31)
+
+
+def random_datetime() -> datetime:
+    return start_datetime + timedelta(
+        seconds=random.randint(0, int((end_datetime - start_datetime).total_seconds())),
+        microseconds=random.randint(0, 999999),
+    )
```

Comment:
I am somewhat skeptical when dealing with datetimes, I find them really fragile things to work with
Could you generate more datetime variants, please?
E.g., with timezones and without, with microseconds and without (ideally for each format we support)

Reply:
How about this

https://github.com/qdrant/qdrant-client/blob/3337b2ae00ff6b8175bd92556fea730e2b0ed66c/tests/fixtures/payload.py#L157-L165

Reply:
sounds good, does not work

Reply:
"%Y-%m-%dT%H:%M:%S.%f%:z"
"%Y-%m-%d %H:%M:%S.%f%:z"
    
Are these actually correct formats?

Reply:
They are, but they seem to be introduced in python 3.12, I'll remove them since we don't actually need them